### PR TITLE
Allow a user to create a business

### PR DESCRIPTION
**Pull request by Euraldius**
* Create header to persist user auth links on all pages
* Create user home page

https://headquarters.io/tinysplash/stories/789

screen shot 2015-02-24 at 1 40 15 pm
screen shot 2015-02-24 at 1 40 08 pm
screen shot 2015-02-24 at 1 40 23 pm
screen shot 2015-02-24 at 1 40 26 pm
screen shot 2015-02-24 at 1 42 32 pm

### CODE REVIEW

File: app/assets/stylesheets/library/animations/_retract.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+@include keyframes(retract) {
+  0% {
+    top: -100%;
+  }
```
Comment (jasonramirez) [inline app/assets/stylesheets/library/animations/_retract.scss]: It is...

File: app/assets/stylesheets/patterns/_navigation.scss [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
 
   .button {
     margin: 1em;
   }
 
+  .landing & {
+    @include position(absolute, 0 0 null 0);
+    border-bottom: none;
+
+    .button {
```
Comment (jasonramirez) [inline app/assets/stylesheets/patterns/_navigation.scss]: This is a temp fix until we split layouts. I am not going to address this.

File: app/assets/stylesheets/library/animations/_retract.scss (hunk 1 above)
Comment (Euraldius) [inline app/assets/stylesheets/library/animations/_retract.scss]: Lol :dog:

**Comment by Euraldius** [comment]
LGTM

**Comment by jasonramirez** [comment]
Bueno! Go ahead and merge it :hamburger:

**Pull request merged**